.github/workflows/pr.yaml

```diff
@@ -224,6 +224,11 @@ jobs:
       - name: Run Unit Tests
         run: |
           make test-unit
+      - name: Upload coverage reports to Codecov
+        uses: codecov/codecov-action@v4.0.1
+        with:
+          token: ${{ secrets.CODECOV_TOKEN }}
```

Comment:
Is this secret already set?

Reply:
@yanivagman to confirm.

Reply:
Changes are already tested on fork repo https://github.com/rahul2393/tracee/pull/6